app/views/local_transactions/_devolved_administrations.html.erb

```diff
@@ -0,0 +1,35 @@
+<% administrations = {
+  "Scotland" => :scotland_availability,
+  "Wales" => :wales_availability,
+  "Northern Ireland" => :northern_ireland_availability
+} %>
+<% administrations.each do |title, field_name| %>
+  <%= f.fields_for field_name do |f| %>
+    <p class="h4 add-top-margin"><%= title %></p>
+
+    <%= form_group(@resource.errors, field_name, classes: %w[radio]) do %>
+      <%= f.label :type_local_authority_service, class: "control-label" do %>
+        <%= f.radio_button :type, 'local_authority_service', disabled: @resource.locked_for_edits? %>
+        Service available from local council
+      <% end %>
+    <% end %>
```

Comment:
I don't think it's worth putting these into the form group since they're already pretty custom and there couldn't be an error on them (as the radio has a value that isn't a field). I'd suggest going back to:

```html
<div class="form-group">
  <div class="radio">
    <%= f.label :type_local_authority_service, class: "control-label" do %>
    <%= f.radio_button :type, 'local_authority_service', disabled: @resource.locked_for_edits? %>
          Service available from local council
    <% end %>
  </div>
</div>
```

Reply:
Pragmatically I totally agree and I'm happy to change these back.  But just for completeness... Technically, there can be an error generated here as the [model validates the type here](https://github.com/alphagov/publisher/blob/3e928a81d339387f91699ba6fba8c4a4d1eb5231/app/models/devolved_administration_availability.rb#L8).  In order to fire this validation error the user needs to 'change' the `type` value on the selected radio via dev tools or similar, then submit the form.  Of course this is not an easy (or obvious) thing to do and I don't think we'd expect a user of an internal application like this to actually do it.

Reply:
Yup, I was thinking (incorrectly) that the form group was looking at a field name of `:type_local_authority_service` which can't have an error (since any errors would be on `:type` as you suggest). I see in the code though that the field name for these groups is `:wales_availability` etc so I think that would cause a strange situation where you could any errors for `errors[:wales_availability]` repeated on every single radio button? That'd not make sense as we'd only want an error specific to that radio button - we'd probably want a spot below the radio buttons to show any errors on `:type` if we thought they'd add value.

This perhaps explains to me why you were deleting "is invalid" as I guess you'd get that under every radio button with this code?

Reply:
Yes, I've just tried it - with- and without-JS - not removing `is invalid` and changing the type value on Northern Ireland to be `unknown` via dev tools.  With-JS is ok, but without-JS...

### With-JS
![with-js](https://user-images.githubusercontent.com/44037625/127462845-9f77f96e-1a6d-4250-bb08-d0075a8f26bd.png)

### Without-JS
![without-js](https://user-images.githubusercontent.com/44037625/127462870-081fcbd8-1e0a-4870-8a09-d33c328ee1c0.png)

Reply:
Have reverted the radio button inputs as suggested.